### PR TITLE
TG Sprite Update Project: Toolboxes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
It's been discussed on and off for over a year now; we're finally going to do it.

This is the beginning of updating our sprite-set to TG's current standard.

This sprite set originated from Goon, managed to make its way to TG, then Bay and pretty much everyone else ended up directly porting from TG for this sprite set. We'll be following suit. This is not merely a sprite update project, but also a refactor to *finally* clean up the utter mess that is our icon files and especially our mob in-hand files, as well.

For more details, please see the document posted in development-news on the Discord.

We'll be starting out with toolboxes, perhaps one of the most iconic sprite in all of SS13, as the first sprites to get changed---what better way to start off a re-spriting project?

img

other: removes the brass toolbox; it's completely unused; clockcult is dead.

:cl: Fox McCloud
imageadd: Updates the toolbox sprites
/:cl:

### CODE REVIEW

**Comment by SabreML** [comment]
While these sprites are objectively better looking than our current ones, I worry about the direction this could take the server.
Most of our 'New features' are just TG ports, we're changing the backend code to be more similar to TG's systems, sound effects are being brought in from TG, and now sprites too?

I just can't help wondering, when's the rebase going to happen?

**Comment by fludd12** [comment]
why not

**Comment by FreeStylaLT** [comment]
https://user-images.githubusercontent.com/11361525/110009150-46cc7b00-7d25-11eb-8017-a5ac7b4224be.mp4

**Comment by SabreML** [comment]
> why not

TG has a *lot* of good features and mechanics, and I have no issues with them being ported over. It'd be a bit hypocritical if I did.
The thing I worry about is that these changes will be added to Paradise without much thought because "Well it's from TG, so clearly it's a good feature to have."

Paradise is a very different server in a number of ways, and I'm just worried about losing that for the sake of becoming another clone.

**Comment by fludd12** [comment]
@SabreML I was mostly meme-ing, but in all honesty I don't actually mind the idea of becoming a downstream of TG, if not for some of the logistical issues surrounding it. A lot of what makes Paradise what it is rests mostly on the players, as well as our variety of species and those interactions -- and redoing those species under /tg/'s system is fairly easy.

**Comment by Wojz** [comment]
> > I just don't want Paradise to become 'TG downstream # 37'
> 
> why not

because if I wanted to play /tg/ I would, and with the current tides bringing in new players we've already started to lose identity

**Comment by Ionward** [comment]
I think worrying that sprite changes would change any server rules or standards is pretty silly.

**Comment by Varghh** [comment]
I have one question, what was the reason of the sprite contest @McRamon won if were just going to port everything from tg? I love new sprites, I much ratter have them made by Ramon instead of just porting everything.

**Review by Kyep — APPROVED** [review]
I'll sign off on this, simply because I feel the new sprites look good.
Like everything else, you have to review each proposed change on a case by case basis.

**Comment by Necaladun** [comment]
Yeah, let's bring us into the 2010s.

@SabreML 
We're not going to be porting stuff without considering the effects or balance simply because it's from TG. We've been facing this accusation for years, despite rejecting huge amounts of features and other things from TG. Never have or will we merge something because "Well it's from TG, so clearly it's a good feature to have.". That's a strawman concern at best.

**Review by AffectedArc07 — APPROVED** [review]
Just so everyone is aware. Changing sprites is not a rebase. I myself am **against** an entire rebase to TG. Their art direction does look nicer (imo), not to mention its actually a consistent art direction, compared to our current hodgepodge of perspective.
image

Just in this image here, you can see disposals and SSUs are face on, lockers at an angle and consoles at an angle but not quite so.

**Comment by Spacemanspark** [comment]
Oh huh I'm not the only one then

I'll stick my own thoughts in that the sprites aren't bad by a long shot--but I'd rather that we stopped porting all of our features and instead focused on making our own. Something that puts us apart from other servers and gives us more of an identity. Having ports here and there is completely fine, but when it becomes all of our content... why not just rebase and restructure the entirety of what the server is at this point? 

You can look at this on a case by case situation, perhaps, but the overall thing with these ports paints a picture about us that I'm not terribly keen on. And that's important to consider.

I'd love to see Ramon make new sprites for Paradise, things that give us a bit more unique flavor and set us apart from other servers. Community is certainly very important and I'm glad we have a somewhat different player ideology than other servers--but our features and mechanics are also important. It's what gives our players the setting, and the tools they need to immerse themselves within it.

**Comment by Fox-McCloud** [comment]
So it begins.

**Comment by Kyep** [comment]
> TG has a _lot_ of good features and mechanics, and I have no issues with them being ported over. It'd be a bit hypocritical if I did.
> The thing I worry about is that these changes will be added to Paradise without much thought because "Well it's from TG, so clearly it's a good feature to have."
> 
> Paradise is a very different server in a number of ways, and I'm just worried about losing that for the sake of becoming another clone.
> Some reassurance or ~any~ communication would be good for this, I think.

@SabreML 
Speaking only for myself here:
I won't hesitate to object to a PR that I feel isn't a good fit for Paradise, regardless of whether its based on TG, another server, or is original work. If I do object to a PR, I will post a review explaining why, so the PR author is clearly informed what the issue is.
I treat stuff from TG with *more* skepticism than I treat original made-for-Paradise work. Partly because I want us, as a server, to have unique content, and partly because I'm well aware we have a history of porting stuff from TG without adequately adjusting it for Paradise's playstyle/balance.
I'm also by far the most active head when it comes to voting yes/no on PRs. To give an example, as of writing, of all head votes to approve PRs on our internal tracker, 75% of all those head votes came from me.
In short: I already go to some extra effort to make sure we're not thoughtlessly copying stuff from TG. It is something I guard against. HOWEVER, that doesn't mean that any port from TG is bad. Each port has to be considered on its own merits. Dismissing everything from TG is just as bad as accepting everything from TG.

**Comment by Spacemanspark** [comment]
I don't think anyone is saying dismissing everything from tg is a bad idea. They _do_ have some really cool snit.

Just that the constant ports are a bit wearing/ concerning.

**Comment by Kyep** [comment]
@Spacemanspark 
I agree, constant ports from TG are wearing/concerning, and I'll be happy to object to any of them that I do not feel are better *for Paradise* than what we already have.
However, there are some of them (like this one) that make sense.
It is a case-by-case thing.
More broadly, I do want us to make a bigger effort to make original content - I feel we are sorely lacking in original content, and we need to make a much better effort to support it instead of just relying on ports. That, however, is a discussion outside the scope of this PR.

**Comment by Spacemanspark** [comment]
I would be a bit less quick to bring it up in a place such as here if the actual areas for this very same discussion were having attention shown to them as well. There's a very particular forum post that I've noticed, which has absolutely zero proper discussion outside of a long reply from me and a few small statements from one or two others.

When I see others actually talking about it in a non hostile way (I'm rather happy people aren't being toxic about it here--thanks nerds) I feel a lot more compelled to voice my own concerns alongside them, where I feel ill be seen and/ or heard.
